src/posts/paper/14-research-sources.ts

```diff
@@ -330,11 +333,10 @@ Data from [Moderna’s phase 3 study](https://www.nejm.org/doi/full/10.1056/NEJM
 | Symptomatic Cases | 185           | 11               |
 | Severe Cases      | 30            | 0                |
 
-First, we note that these two vaccines are very, very effective at preventing symptomatic COVID (95% for Pfizer and 94% for Moderna). Second, we note that the Phase III studies do not include data on cases without symptoms. 
 
-However, in a [supplement provided to the FDA](https://www.fda.gov/media/144453/download), Moderna showed a 63% reduction in positive PCR swabs at the time of the 2nd dose among people who had no symptoms of COVID up to that point.
+Fortunately, we have access to a [supplement provided to the FDA](https://www.fda.gov/media/144453/download), which provides a useful clue about reduction of non-symptomatic cases. In this supplementary data, Moderna showed a 63% reduction in positive PCR swabs at the time of the 2nd dose among people who had no symptoms of COVID up to that point. Some of these people might've later shown symptoms. This indicates that the total reduction in Later-Symptomatic + Never Symptomatic COVID cases from *the first dose* of Moderna’s vaccine was 63%.
 
-This indicates that the total reduction in Symptomatic + Never Symptomatic COVID cases from *the first dose* of Moderna’s vaccine is 63%. Likely it is even better after the second dose. From this, we construct the following chart:
+We therefore use \`1 - 0.63 = 0.37 \` as a conservative estimate of the reduction of total COVID cases in the vaccinated group. Likely it is even better after the second dose. From this, we construct the following chart:
```

Comment:
should we mention that there are considerable error bars on using data from a single day snapshot like this?